### ISSUE TITLE
Windows List bar detached from main window

### ISSUE THREAD

**Issue by mike362** [pawelsalawa/sqlitestudio#4326]
### Details
I found that the Windows List bar, by default displayed at the bottom of the main window can be moved and repositioned to the left of the main window by clicking and holding a 'handle' on the left end of the bar. Whilst moving it around I accidentally released my mouse button at the left edge of the screen, the bar is now permanently set there and I can't see or click on the handle to move it again. Using a dual screen set up, if I move the main window to my other screen the bar remains on the the original screen locked in place, it is detached from the main window. I've triedto find something in config to and background setting to reset the bar position without luck. I've downloaded a fresh copy of the portable app and fired that up with the same problem so the settings must be recorded outside the portable app directory.

Solution: Perhaps there can be a 'Reset Defaults' option in the config dialog window that restores all of the original settings? For the portable app, can any settings not held in the SQLiteStudio root or child directories be moved there to make an installation independent from other installations?

### Steps to reproduce
Click and hold on the 'handle' at the left end of the Window List bar and drag off the left side of the screen and release.

### Operating system
Windows 10 1909

### SQLiteStudio version
3.3.3

**Comment by pawelsalawa** [comment]
When I move the bar out of window and leave it there, it still has the handle:
![image](https://user-images.githubusercontent.com/481846/144677874-caab1483-e32d-47b2-8939-b02907015786.png)

Doesn't it for you?

If I grab that handle and move it back to the window, I'm able to re-attach it:
![image](https://user-images.githubusercontent.com/481846/144677927-e75b6f5e-3e3a-4c2d-9bed-ae023a0412a8.png)

**Comment by mike362** [comment]
This is what I see with the Windows List bar off the left hand edge of my screen.

![image](https://user-images.githubusercontent.com/46487901/144718223-33757c26-e9c6-469a-9690-7ff65e837c5a.png)

The handle at the left end of the toolbar is just off the screen so I can't click and drag with it.

I've dug around my file system and find a settings file at C:\Users\_myusername_\AppData\Local\SalSoft\SQLiteStudio, the file is settings3. I've found that if I rename it a fresh version is created when I open SqliteStudio with the default settings so the Windows List bar is back in place at the bottom of the screen. Replacing that with the version of settings3 where I had the problem recreates the problem so I've attached it, zipped using IZArc, to see if it creates the problem for you. I don't see which setting would record the position of the bar in settings3 to try to manually change it.

Thanks for your help
[settings3.zip](https://github.com/pawelsalawa/sqlitestudio/files/7654321/settings3.zip)
![SqliteStudio Windows List bar](https://user-images.githubusercontent.com/46487901/144718507-e02389ff-e5a3-4612-b4dd-dea0c2e84989.png)
.
